src/components/field-key/qr-panel.vue

```diff
@@ -29,13 +29,13 @@ except according to the terms contained in the LICENSE file.
           </template>
         </i18n-t>
         <sentence-separator/>
-        <template v-if="managed">{{ $t('body[1].managed', fieldKey) }}</template>
+        <template v-if="showManaged">{{ $t('body[1].managed', fieldKey) }}</template>
         <template v-else>{{ $t('body[1].legacy') }}</template>
         <sentence-separator/>
-        <i18n-t v-if="managed" keypath="body[2].managed.full">
+        <i18n-t v-if="showManaged" keypath="body[2].managed.full">
           <template #switchToLegacy>
             <i18n-t tag="a" keypath="body[2].managed.switchToLegacy"
-              class="switch-code" href="#">
+              class="switch-code" href="#" @click.prevent="toggleManaged">
```

Comment:
Could emit a `toggleManaged` event and the parent component could toggle `managed` from the outside

Reply:
Could also use the provide/inject mechanism for a ref to more easily change legacy/managed here. Not reactive by default so it would have to explicitly provide a reactive ref.